src/mozilla_sec_eia/models/sec10k/notebooks/exhibit21_layout_classifier.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        def predict(self, context, model_input: pd.DataFrame):
</pre><p>I assume we use predict when we pull down the classifier during an inference run and actually classify documents?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/mozilla-sec-eia/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>